lib/ecmascript.ts

```diff
@@ -147,6 +148,19 @@ function ToIntegerNoFraction(valueParam: unknown): number {
   }
   return value;
 }
+function divmod(x: JSBI, y: JSBI): { quotient: JSBI; remainder: JSBI } {
```

Comment:
I don't have a strong opinion one way or the other, but it may be good to just inline this where it's used? I'm not sure if the quotient and remainder are both used in every instance.

Reply:
I avoided inlining this because it makes the diffs much easier to read in some cases where there are several divmod operations happening sequentially.